e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -94,40 +113,37 @@ describe("banner", () => {
   });
 });
 
-describeWithSnowplow(
-  "should send snowplow events when clicking on links in the database prompt banner",
-  () => {
-    const PAGE_VIEW_EVENT = 1;
-
-    beforeEach(() => {
-      restore();
-      resetSnowplow();
-      cy.signInAsAdmin();
-      enableTracking();
-      cy.visit("/");
-      cy.findByRole("main").findByText("Loading...").should("not.exist");
-    });
+describeWithSnowplow("database prompt banner", () => {
```

Comment:
I only change the name on this line and in the `it()` line below, but Prettier formatting makes this looks like a huge diff.

I need to make the name change because the test name in `it()` line was completely wrong.

Reply:
Dang, that's dope. I never know that.